src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/ViewSpecificMethodologyAuthorizationHandlers.cs

```diff
@@ -1,28 +1,53 @@
-using GovUk.Education.ExploreEducationStatistics.Common.Security.AuthorizationHandlers;
+using System.Dynamic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Methodologies;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Security;
 using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
 using Microsoft.AspNetCore.Authorization;
+using Microsoft.EntityFrameworkCore;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Security.AuthorizationHandlers
 {
     public class ViewSpecificMethodologyRequirement : IAuthorizationRequirement
     {}
 
-    public class ViewSpecificMethodologyAuthorizationHandler : CompoundAuthorizationHandler<ViewSpecificMethodologyRequirement, Methodology>
+    public class ViewSpecificMethodologyAuthorizationHandler
+        : CompoundAuthorizationHandler<ViewSpecificMethodologyRequirement, Methodology>
     {
-        public ViewSpecificMethodologyAuthorizationHandler() : base(
-            new CanViewAllMethodologiesAuthorizationHandler())
-        {
-            
-        }
+        public ViewSpecificMethodologyAuthorizationHandler(IMethodologyRepository methodologyRepository) : base(
+            new CanViewAllMethodologiesAuthorizationHandler(),
+            new HasRoleOnAnyAssociatedReleaseAuthorizationHandler(methodologyRepository)) {}
     
         public class CanViewAllMethodologiesAuthorizationHandler 
             : HasClaimAuthorizationHandler<ViewSpecificMethodologyRequirement>
         {
             public CanViewAllMethodologiesAuthorizationHandler()
-                : base(SecurityClaimTypes.AccessAllMethodologies)
+                : base(SecurityClaimTypes.AccessAllMethodologies) {}
+        }
+
+        public class HasRoleOnAnyAssociatedReleaseAuthorizationHandler
```

Comment:
Same as in `UpdateSpecificMethodologyAuthorizationHandler`, would consider making these dependent handlers nested and dropping the `AuthorizationHandler` part of the name!